### ISSUE TITLE
No debug controls on macOS touch bar when editor active

### ISSUE THREAD

**Issue by goda** [microsoft/vscode#106948]
- VSCode Version: 1.49.0
- OS Version: macOS Catalina 10.15.5

Steps to Reproduce:

1. Start a debug session and place breakpoint in code
2. When execution stops at breakpoint, Touch Bar no longer shows the debug controls.

Does this issue occur when all extensions are disabled?: N/A can't try this - need .NET Test extension to debug application

When I focus on any other part of the window (i.e. side bar) the Touch Bar debug controls show up correctly. There are resolved issues on this from 2019 but I never had this issue until I upgraded to 1.49.0 this week.

**Comment by goda** [comment]
Bit more info,

This is the state of the Touch Bar when editor is active:
<img width="1085" alt="image" src="https://user-images.githubusercontent.com/480738/93522577-b2e62e80-f8ff-11ea-9668-80689a8383a4.png">

and this is the state when the editor is not active:
<img width="1085" alt="image" src="https://user-images.githubusercontent.com/480738/93522634-c85b5880-f8ff-11ea-9de2-65628b7236b4.png">

One thing I've noticed is that if I click on the title window bar, while the editor is active, the Touch Bar flashes and briefly shows the debug controls as in second image before reverting to the first.

I tried the Insiders version and same issue is present there.

Hope that's helpful in debugging.
